test/runtests.bat

```diff
@@ -13,4 +13,6 @@ REM run the program, feed commands from input.txt file and redirect the output t
 java -classpath ..\bin seedu.addressbook.Main < input.txt > actual.txt
 
 REM compare the output to the expected output
-FC actual.txt expected.txt
\ No newline at end of file
+FC actual.txt expected.txt
```

Comment:
This change should not be in this PR. Check the staged changes before committing to ensure it doesn't contain unintended changes. You can use Git's ignore feature to ignore files that should not be version tracked.